database/sql/src/main/scala/cromwell/database/sql/MetadataSqlDatabase.scala

```diff
@@ -185,7 +185,7 @@ trait MetadataSqlDatabase extends SqlDatabase {
 
   def getSummaryQueueSize()(implicit ec: ExecutionContext): Future[Int]
 
-  def getMetadataArchiveStatus(workflowId: String)(implicit ec: ExecutionContext): Future[Option[String]]
+  def getMetadataArchiveStatusAndEndTime(workflowId: String)(implicit ec: ExecutionContext): Future[(Option[String], Option[Timestamp])]
```

Comment:
I'm not wild about introducing a widely used tuple of slightly unclear meaning.

Could we create a `MetadataArchiveSummary` or similar case class that includes status and end timestamp?

Reply:
👍  in general although at _this_ level (`MetadataSqlDatabase`) I believe we try to keep it "primitives only". So I added the requested case class but just up at the `MetadataDatabaseAccess` level.